frontend/karma.conf.js

```diff
@@ -16,7 +16,8 @@ module.exports = function (config) {
     plugins: [
       require('karma-jasmine'),
       require('karma-chrome-launcher'),
-      require('@angular-devkit/build-angular/plugins/karma')
+      require('@angular-devkit/build-angular/plugins/karma'),
+      require('karma-spec-reporter'),
```

Comment:
I replaced the default formatter to output the specs being run for clarity which fails